liquibase-core/src/main/java/liquibase/statement/ExecutablePreparedStatementBase.java

```diff
@@ -130,6 +131,8 @@ private void applyColumnParameter(PreparedStatement stmt, int i, ColumnConfig co
             LOG.debug(LogType.LOG, "value is string = " + col.getValue());
             if (col.getType() != null && col.getType().equalsIgnoreCase(LoadDataChange.LOAD_DATA_TYPE.UUID.name())) {
                 stmt.setObject(i, UUID.fromString(col.getValue()));
+            } else if (col.getType() != null && col.getType().equalsIgnoreCase(LoadDataChange.LOAD_DATA_TYPE.OTHER.name())) {
+                stmt.setObject(i, col.getValue(), Types.OTHER);
```

Comment:
Is this code take care of JSON objects as well ?

Reply:
I'm not sure if we need this for Json columns. If I remember correctly (I'm not associated with the organization for which I wrote this), I was already using STRING for the json columns and it was working just fine.